public/index.html

```diff
@@ -7,36 +7,234 @@
+            See projects
+        </button>
+    </section>
+
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/2.png" alt="">
+        <h2>Multi-Post
+            Stories</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/3.png" alt="">
+        <h2>Tonic</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/4.png" alt="">
+        <h2>Multi-Post
+            Stories</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
```

Comment:
- [ ] Please consider wrapping all these project items into one section, which is the work section that will use `grid` to layout all work cards. Also consider to make sure that all these project items (`card-wrapper`) should also not use `<section>` but `li` and should be wrapped in a `ul`. This will make your code look much cleaner 🧼 and maintainable 😄